core/cgi.ml

```diff
@@ -0,0 +1,328 @@
+(*
```

Comment:
It seems `cgi.ml` has been readded. I thought we had gotten rid of it.

Reply:
See P2) in my PR description:  tldr: The removal of cgi isn't merged in yet, so I patched cgi as an immediate solution to work with 4.06.0